packages/perseus/src/widgets/categorizer/categorizer-validator.test.ts

```diff
@@ -7,17 +7,7 @@ import type {PerseusCategorizerRubric} from "../../validation.types";
 describe("categorizerValidator", () => {
     it("gives points when the answer is correct", () => {
         const rubric: PerseusCategorizerRubric = {
-            items: ["Graph $1$", "Graph $2$"],
```

Comment:
It's nice to see how much clearer this makes the types but differentiating (1) what we need for rendering vs (2) what we need from the learner vs (3) what we need to score. Love to see this.

Reply:
Very much agree! This would be extremely useful even if we weren't doing SSS.